packages/smooth_app/lib/main.dart

```diff
@@ -21,16 +21,18 @@ import 'package:smooth_ui_library/navigation/smooth_navigation_layout.dart';
 import 'package:provider/provider.dart';
 import 'package:smooth_app/temp/user_preferences.dart';
 
-final SentryClient sentry = SentryClient(
-    dsn:
-        'https://22ec5d0489534b91ba455462d3736680@o241488.ingest.sentry.io/5376745');
-
 Future<void> main() async {
+  await Sentry.init(
```

Comment:
I guess this Sentry part should be part of a separate PR related to #9. This PR is about dark mode.

Reply:
I just thought that for the small change it is unnecessary to open a new pr

Reply:
I understand, but that's a source of confusion between issues, and there's no obvious sign that you fully tested the Sentry part. Perhaps other contributors that know things about Sentry and nothing about dark mode would like to make suggestions about Sentry, but it will be hidden "in the dark" ;)
That definitely makes sense to create a specific PR for Sentry, and you even already have a specific issue for that (#9).